curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/inpaint.html

```diff
@@ -0,0 +1,129 @@
+<html>
+<head>
+<meta charset="utf-8">
+<title>CVdrawing</title>
+</head>
+<body>
+<p id="status">OpenCV.js is loading...</p>
+<canvas id="canvas" width="640" height="426">
+</canvas>
+<img id="src" src="bird.jpg" style="display:none"/>
+<input type="button" id="myButton" value="Test" disabled=true
+onclick="inpaint()"/>
+
+<script type="text/javascript">
+
+var checker = false;
+var src;
+var mask;
+var dst;
+var p_x = 0;
+var p_y = 0;
+var paint;
+var srcmat;
+var maskmat;
+var dstmat;
+var x;
+var y;
+
+window.onload = function()
+{
+context = document.getElementById("canvas").getContext("2d");
+img = document.getElementById("src");
+context.drawImage(img, 0, 0);
+};
```

Comment:
imread + imshow?

Reply:
We need that function for drawing in canvas, can't replace them with OpenCV functions.

Reply:
Have you tried? As far as I remember only imshow for <img> does not work. Here is imread from <img>  and imshow to <canvas>

Reply:
We can not draw on cv.Mat(). cv.Mat() is not a constructor, we have in that case type error. I tested that option earlier.
We need canvas for drawing in real time.

Reply:
Can you try to test this method:
```js
window.onload = function()
{
    let img = cv.imread("src");
    cv.imshow("canvas", img);
};
```

Reply:
I tried that case - we got here : "Uncaught TypeError: cv.Mat is not a constructor".

Reply:
@ilbalance, Got it, thanks! It about initialization of OpenCV - we cannot use `cv.*` methods before `onRuntimeInitialized`.